### PR TITLE
Fix for UUID

### PR DESCRIPTION
**Pull request by netcreatoreu**
FOSElasticaBundle doesn't work correctly when we use UUID as primary key.

### CODE REVIEW

**Comment by merk** [comment]
An identifier should not be locked into being a string. 

Does your UUID class have a __toString() method? Otherwise, add a method 'getIdentifierString' and change the property path for the identifier in your type configuration.

**Comment by damienalexandre** [comment]
Keep in mind that identifier are always strings in Elasticsearch.

File: composer.json [hunk 1]
```diff
@@ -11,13 +11,19 @@
         { "name": "Richard Miller", "email": "richard.miller@limethinking.co.uk" },
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com" }
     ],
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/netcreatoreu/Elastica"
```
Comment (solivier) [inline composer.json]: I don't think this should be merged

**Comment by merk** [comment]
If ES identifiers are always strings then this isnt such a big deal. Since #1067 has test cases I'm happy to merge that one.

**Pull request closed without merging**